README.md

```diff
@@ -49,8 +49,8 @@
 - 🇪🇸 Spanish
 - 🇸🇪 Swedish
 - 🇮🇳 Tamil
-- ᴛʜ Thai
-- ᴠ🇳 Vietnamese
+- 🇹🇭 Thai
```

Comment:
There is already an open pull request for fixing the Thailand flag. If your pull request gets merged before I will decline mine.